sk0-front/src/app/global-services/localstore.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from "@angular/core";
+
+@Injectable({
+  providedIn: "root"
+})
+export class LocalStoreService {
+  constructor() {}
```

Comment:
If your constructor is empty you can delete it. (And we can re-add whenever we will need it.